File: api/src/main/java/hirez/api/Portal.java
package hirez.api;

import hirez.api.object.interfaces.IDObject;

public enum Portal implements IDObject<Integer> {
    UNKNOWN(-1),
    HIREZ(1),
    STEAM(5),
    PS4(9),
    XBOX(10),
    SWITCH(22),
    DISCORD(25);

    private final Integer id;

    Portal(int id) {
        this.id = id;
    }

    @Override
    public Integer getId() {
        return id;
    }
}

File: api/src/main/kotlin/hirez/enums/Portal.kt
package hirez.enums

/**
 *
 * @author Damian Staszewski [damian@stachuofficial.tv]
 * @version %I%, %G%
 * @since 1.0
 */
enum class Portal(val id: Int) {
	UNKNOWN(-1),
	HIREZ(1),
	STEAM(5),
	PS4(9),
	XBOX(10),
	SWITCH(22)
}